Document 'default_group_priority' and 'supervisor' priority (#944)
livechat/livechat-public-docs: content/management/configuration-api/index.mdx

| `first`        |                                                           The highest chat routing priority. Agents with the `first` priority get chats before others from the same group, e.g. Bots can get chats before regular Agents.                                                           |
| `normal`       | The medium chat routing priority. Agents with the `normal` priority get chats before those with the `last` priority, when there are no Agents with the `first` priority available with free slots in the group. Corresponds to the **Primary** priority option in the LiveChat App. |
| `last`         |              The lowest chat routing priority. Agents with the `last` priority get chats when there are no Agents with the `first` or `normal` priority available with free slots in the group. Corresponds to the **Secondary** priority option in the LiveChat App.               |

> | `supervisor`   |                                                                                                         Agent works as `supervisor` so it will not be assigned to any chats.                                                                                                        |

Reviewer: ```suggestion
| `supervisor`   |                                                                                                         Agents with the `supervisor` priority will not get any chats assigned automatically.                                                                                                        |
```

| `first`        |                                                           The highest chat routing priority. Agents with the `first` priority get chats before others from the same group, e.g. Bots can get chats before regular Agents.                                                           |
| `normal`       | The medium chat routing priority. Agents with the `normal` priority get chats before those with the `last` priority, when there are no Agents with the `first` priority available with free slots in the group. Corresponds to the **Primary** priority option in the LiveChat App. |
| `last`         |              The lowest chat routing priority. Agents with the `last` priority get chats when there are no Agents with the `first` or `normal` priority available with free slots in the group. Corresponds to the **Secondary** priority option in the LiveChat App.               |

> | `supervisor`   |                                                                                                         Agent works as `supervisor` so it will not be assigned to any chats.                                                                                                        |

Reviewer: ```suggestion
| `supervisor`   |                                                                                                         Agents with the `supervisor` priority will not get any chats assigned automatically.                                                                                                        |
```

| Possible values |                                                                                                      Notes                                                                                                      |
| --------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| `first`         |                         The highest chat routing priority. Agents with the `first` priority get chats before others from the same group, e.g. Bots can get chats before regular Agents.                         |
| `normal`        | The medium chat routing priority. Agents with the `normal` priority get chats before those with the `last` priority, when there are no Agents with the `first` priority available with free slots in the group. |
| `last`          |               The lowest chat routing priority. Agents with the `last` priority get chats when there are no Agents with the `first` or `normal` priority available with free slots in the group.                |

> | `supervisor`    |                                                                       Bot works as `supervisor` so it will not be assigned to any chats.                                                                        |

Reviewer: As I see we were consistent with the possible values being the same as for Agents for the Bot-related values as well 🤔  I _think_ we should stick to that and keep:
```suggestion
| `supervisor`    |                                                                       Agents with the `supervisor` priority will not get any chats assigned automatically.                                                                        |
```
but I will need to consult this with Ola and let you know :)

| `first`         |                         The highest chat routing priority. Agents with the `first` priority get chats before others from the same group, e.g. Bots can get chats before regular Agents.                         |
| `normal`        | The medium chat routing priority. Agents with the `normal` priority get chats before those with the `last` priority, when there are no Agents with the `first` priority available with free slots in the group. |
| `last`          |               The lowest chat routing priority. Agents with the `last` priority get chats when there are no Agents with the `first` or `normal` priority available with free slots in the group.                |

> | `supervisor`    |                                                                       Bot works as `supervisor` so it will not be assigned to any chats.                                                                        |

Reviewer: the same as for the above - will check with Ola

| Possible values |                                                                                                      Notes                                                                                                      |
| --------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| `first`         |                         The highest chat routing priority. Agents with the `first` priority get chats before others from the same group, e.g. Bots can get chats before regular Agents.                         |
| `normal`        | The medium chat routing priority. Agents with the `normal` priority get chats before those with the `last` priority, when there are no Agents with the `first` priority available with free slots in the group. |
| `last`          |               The lowest chat routing priority. Agents with the `last` priority get chats when there are no Agents with the `first` or `normal` priority available with free slots in the group.                |

> | `supervisor`    |                                                                       Bot works as `supervisor` so it will not be assigned to any chats.                                                                        |

Reviewer: and same here :)
